lib/device.jl

```diff
@@ -24,7 +24,7 @@ end
         clGetDeviceInfo(d, prop, 0, C_NULL, sz)
         chars = Vector{Cchar}(undef, sz[])
         clGetDeviceInfo(d, prop, sz[], chars, C_NULL)
-        return CLString(chars)
+        return unsafe_string(pointer(chars))
```

Comment:
I believe this is unsafe without preserving the Vector before it gets converted to String.

Reply:
Thanks; https://github.com/JuliaGPU/OpenCL.jl/pull/219